extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -301,18 +301,14 @@ describe('amp-a4a', () => {
   /**
    * @param {!AmpA4A} a4a
    * @param {!Function} triggerAnalyticsEventSpy
+   * @param {(string|Array<string>)=} additionalEvents
    */
-  function verifyA4aAnalyticsTriggersWereFired(a4a, triggerAnalyticsEventSpy) {
-    expect(triggerAnalyticsEventSpy).to.be.calledWith(
-        a4a.element, 'ad-request-start', {'time': sinon.match.number});
-    expect(triggerAnalyticsEventSpy).to.be.calledWith(
-        a4a.element, 'ad-response-end', {'time': sinon.match.number});
-    expect(triggerAnalyticsEventSpy).to.be.calledWith(
-        a4a.element, 'ad-render-start', {'time': sinon.match.number});
-    expect(triggerAnalyticsEventSpy).to.be.calledWith(
-        a4a.element, 'ad-render-end', {'time': sinon.match.number});
-    expect(triggerAnalyticsEventSpy).to.be.calledWith(
-        a4a.element, 'ad-iframe-loaded', {'time': sinon.match.number});
+  function verifyA4aAnalyticsTriggersWereFired(
+    a4a, triggerAnalyticsEventSpy, additionalEvents = []) {
+    ['ad-request-start', 'ad-response-end', 'ad-render-start',
+      'ad-render-end', 'ad-iframe-loaded'].concat(additionalEvents)
```

Comment:
For extra ES6-ness, you can replace `concat` with the spread operator:

```
['ad-request-start', 'ad-response-end', 'ad-render-start',
   'ad-render-end', 'ad-iframe-loaded', ...additionalEvents]
```

Reply:
AMP generally discourages the spread operator (see https://github.com/ampproject/amphtml/blob/master/build-system/eslint-rules/no-spread.js)